js/frames/packJapanShowcase.js

```diff
@@ -0,0 +1,60 @@
+//Create objects for common properties across available frames
+var masks = [{src:'/img/frames/m15/japaneseShowcase/japaneseShowcaseMaskPinline.png', name:'Pinline'}, {src:'/img/frames/m15/regular/m15MaskTitle.png', name:'Title'}, {src:'/img/frames/m15/regular/m15MaskType.png', name:'Type'}, {src:'/img/frames/m15/regular/m15MaskRules.png', name:'Rules'}, {src:'/img/frames/m15/regular/m15MaskBorder.png', name:'Border'}];
+var bounds = {x:0.7771, y:0.8876, width:0.1720, height:0.0593};
+var boundsStamp = {x:0.4365, y:0.902, width:0.1264, height:0.0452};
+//defines available frames
+availableFrames = [
+	{name:'White Frame', src:'/img/frames/m15/japaneseShowcase/japaneseShowcaseFrameW.png', masks:masks},
```

Comment:
Can we make the frame file names be just "w.png", "u.png", etc?
P/T and Holo Stamps can go in their own subdirectory, so the path would be /pt/u.png, and stamps would be /stamp/u.png, for example.
I have been using this naming scheme for my new frames so that they can easily be added to the Autoframe system in the future.